core/src/ids.rs

```diff
@@ -193,6 +193,9 @@ impl MessageId {
 declare_id!(ProgramId: "Program identifier");
 
 impl ProgramId {
+    /// System program ID
+    pub const SYSTEM: Self = Self([0; HASH_LENGTH]);
```

Comment:
you see, zero address is widely used in tests. What about making system address from b'gear' bytes (8 times, so it's going to be 32 bytes)

Reply:
Done